### PR TITLE
Fixed docker message

### PR DESCRIPTION
**Pull request by jlim13**
changed your docker pull command as a response to issue #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ conda env create -f environment.yml -n studiogan
 
 With docker, you can use:
 ```
-docker pull mgkang/studiogan:latest
```
Comment (burakturkoglu54) [inline README.md:84]: 84

Comment (burakturkoglu54) [inline README.md:84]: 84

File: README.md [hunk 2]
```diff
@@ -228,7 +228,7 @@ FID is a widely used metric to evaluate the performance of a GAN model. Calculat
 
 
 ### Precision and Recall (PR)
-Precision measures how accurately the generator can learn the target distribution. Recall measures how completely the generator covers the target distribution. Like IS and FID, calculating Precision and Recall requires the pre-trained Inception-V3 model. StudioGAN uses the same hyperparameter settings with the [original Precision and Recall implementation](https://github.com/msmsajjadi/precision-recall-distributions), and StudioGAN calculates the F-beta score suggested by [Sajjadi et al](https://arxiv.org/abs/1806.00035). 
+Precision measures how accurately the generator can learn the target distribution. Recall measures how completely the generator covers the target distribution. Like IS and FID, calculating Precision and Recall requires the pre-trained Inception-V3 model. StudioGAN uses the same hyperparameter settings with the [original Precision and Recall implementation](https://github.com/msmsajjadi/precision-recall-distributions), and StudioGAN calculates the F-beta score suggested by [Sajjadi et al](https://arxiv.org/abs/1806.00035).
```
Comment (burakturkoglu54) [inline README.md:231]: 231

File: README.md [hunk 3]
```diff
@@ -262,7 +262,7 @@ We report the best IS, FID, and F_beta values of various GANs.
 | [**ADAGAN(P)**](https://arxiv.org/abs/2006.06676) | 64 | 1024 |  |  |  |  | [Config](./src/configs/CIFAR10/ADAGAN(P).json) | [Log]() | [Link]() |
 | [**ADAGAN(C)**](https://arxiv.org/abs/2006.06676) | 64 | 1024 |  |  |  |  | [Config](./src/configs/CIFAR10/ADAGAN(C).json) | [Log]() | [Link]() |
 | [**LOGAN**](https://arxiv.org/abs/1912.00953) | 32 | 64 | 9.576 | 8.465 | 0.993 | 0.990 | [Config](./src/configs/CIFAR10/LOGAN.json) |  [Log](./logs/CIFAR10/LOGAN-train-2020_09_17_13_46_47.log) | [Link](https://drive.google.com/drive/folders/1E9ST1wnh6_rA2Q1eIjydadeWPiZHufvu?usp=sharing) |
-  
+
```
Comment (burakturkoglu54) [inline README.md:265]:
```suggestion

```

File: README.md [hunk 4]
```diff
@@ -262,7 +262,7 @@ We report the best IS, FID, and F_beta values of various GANs.
 | [**ADAGAN(P)**](https://arxiv.org/abs/2006.06676) | 64 | 1024 |  |  |  |  | [Config](./src/configs/CIFAR10/ADAGAN(P).json) | [Log]() | [Link]() |
 | [**ADAGAN(C)**](https://arxiv.org/abs/2006.06676) | 64 | 1024 |  |  |  |  | [Config](./src/configs/CIFAR10/ADAGAN(C).json) | [Log]() | [Link]() |
 | [**LOGAN**](https://arxiv.org/abs/1912.00953) | 32 | 64 | 9.576 | 8.465 | 0.993 | 0.990 | [Config](./src/configs/CIFAR10/LOGAN.json) |  [Log](./logs/CIFAR10/LOGAN-train-2020_09_17_13_46_47.log) | [Link](https://drive.google.com/drive/folders/1E9ST1wnh6_rA2Q1eIjydadeWPiZHufvu?usp=sharing) |
-
```
Comment (burakturkoglu54) [inline README.md:265]:
```suggestion
  ```suggestion
  ```suggestion
-   []()
```

```

```

File: README.md (hunk 1 above)
Comment (burakturkoglu54) [inline README.md:84]: README.md

Comment (burakturkoglu54) [inline README.md:84]: README.md

Comment (burakturkoglu54) [reply README.md:84]: https://github.com/POSTECH-CVLab/PyTorch-StudioGAN/pull/37#discussion_r587153536

File: README.md (hunk 2 above)
Comment (burakturkoglu54) [reply README.md:231]: README.md

Comment (burakturkoglu54) [reply README.md:231]: README.md

Comment (burakturkoglu54) [reply README.md:231]: README.md

File: README.md (hunk 1 above)
Comment (ssabawissabawi) [reply README.md:84]: Iraq
